language/compiler/ir-to-bytecode/src/context.rs

```diff
@@ -146,7 +146,8 @@ impl<'a> CompiledDependency<'a> {
             .get(ident_str(name.as_inner()).ok()?)
             .and_then(|idx| {
                 let fh = self.function_pool.get(*idx as usize)?;
-                Some(FunctionSignature {
+                // Writing down the full path to workaround a bug in rust analyzer.
+                Some(vm::file_format::FunctionSignature {
```

Comment:
I think I fixed this. It will be going in shortly or is already in master

So you should be able to revert